src/lib/metrics.ts

```diff
@@ -372,28 +375,40 @@ export default class MetricsMonitor {
         });
 
         eventBus.on(events.DB_TIME, ({ store, action, time }) => {
-            dbDuration.labels(store, action).observe(time);
+            dbDuration.labels({ store, action }).observe(time);
         });
 
         eventStore.on(FEATURE_CREATED, ({ featureName, project }) => {
-            featureToggleUpdateTotal
-                .labels(featureName, project, 'n/a', 'n/a')
-                .inc();
+            featureToggleUpdateTotal.increment({
+                toggle: featureName,
+                project,
+                environment: 'n/a',
+                environmentType: 'n/a',
+            });
         });
         eventStore.on(FEATURE_VARIANTS_UPDATED, ({ featureName, project }) => {
-            featureToggleUpdateTotal
-                .labels(featureName, project, 'n/a', 'n/a')
-                .inc();
+            featureToggleUpdateTotal.increment({
+                toggle: featureName,
+                project,
+                environment: 'n/a',
+                environmentType: 'n/a',
+            });
         });
         eventStore.on(FEATURE_METADATA_UPDATED, ({ featureName, project }) => {
-            featureToggleUpdateTotal
-                .labels(featureName, project, 'n/a', 'n/a')
-                .inc();
+            featureToggleUpdateTotal.increment({
+                toggle: featureName,
+                project,
+                environment: 'n/a',
+                environmentType: 'n/a',
+            });
         });
         eventStore.on(FEATURE_UPDATED, ({ featureName, project }) => {
-            featureToggleUpdateTotal
-                .labels(featureName, project, 'default', 'production')
-                .inc();
+            featureToggleUpdateTotal.increment({
+                toggle: featureName,
+                project,
+                environment: 'default',
+                environmentType: 'production',
```

Comment:
I assume these `environment` and `environmentType` values are on purpose? I found it weird that these ones are specified, but not the others.